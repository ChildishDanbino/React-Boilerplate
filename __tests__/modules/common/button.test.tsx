import React from 'react'
import { render } from 'test-utils';
import Button from 'modules/common/button/button';

describe('Button', () => {
    it('Renders Button Component', () => {
        const { asFragment } = render(<Button />);
        expect(asFragment()).toMatchSnapshot();
    });
});
