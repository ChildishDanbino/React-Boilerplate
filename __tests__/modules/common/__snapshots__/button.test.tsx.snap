// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Renders Button Component 1`] = `
<DocumentFragment>
  <button
    class="sc-bdVaJa ieqGcc"
  />
</DocumentFragment>
`;
